### ISSUE TITLE
Please create a Notepad++ plugin ZIP package and become part of the official plugin list

### ISSUE THREAD

**Issue by dinkumoil** [blumu/NppColumnSort#7]
Your _NppColumnSort_ plugin is quite useful, I use it from time to time when the build-in sorting feature of Notepad++ is not sufficient for certain special sorting tasks.

Unfortunately it is not part of the current official plugin list of Notepad++. Since v7.6 Notepad++ has a build-in plugin management system called _Plugins Admin_. The new system requires some changes to plugin ZIP packages and provides its own plugin list. The old external _Plugin Manager_ plugin management system is not supported anymore, it seems like its supporting web sites will be abandoned in the near future as well.

Please create a ZIP package of the 32 bit and 64 bit versions of your plugin and include it into the official plugin list of Notepad++. To obtain information how to do that please read the according chapter of the new [Notepad++ user manual](https://npp-user-manual.org/docs/plugins), especially the sections about _Plugins Admin_ and the plugin list itself.

**Comment by artie-finkelstein** [comment]
I second this request.

**Comment by blumu** [comment]
I agree that this would be desirable. NppColumnSort used to be part of the official Notepad++ plugins list but it looks like the plugin publishing system has changed and is now maintained via a JSON file in a Github repository at https://npp-user-manual.org/docs/plugins/ 
I have not read the doc in full details but it looks that publishing the plugin amounts to editing the JSON files at
https://github.com/notepad-plus-plus/nppPluginList/blob/master/src/pl.x64.json
and https://github.com/notepad-plus-plus/nppPluginList/blob/master/src/pl.x86.json and sending a PR to the notepad++ project.
Not sure if anything needs to change in the NppColumnSort repo itself.  If anyone has spare cycles to submit the required changes to both projects, I'll welcome them.
